### PR TITLE
Fix typos and add comments

### PR DESCRIPTION
**Pull request by justxuewei**
**What this PR does**:

- Fix typos
- Add comments(review wanted)
- Resort import blocks

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: internal/stream/stream.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ package stream
 
 import (
 	"bytes"
+	perrors "github.com/pkg/errors"
```
Comment (AlexStocks) [inline internal/stream/stream.go]: pls format ur imports.

File: internal/message/message.go [hunk 2]
```diff
@@ -60,31 +60,31 @@ func (bm *Message) GetMsgType() MsgType {
 	return bm.MsgType
 }
 
-// MsgChain contain the chan of Message
-type MsgChain struct {
+// MsgChan contain the chan of Message
```
Comment (AlexStocks) [inline internal/message/message.go]: maybe the name MsgQueue is more perfect.

Comment (xavier-niu) [reply internal/message/message.go]: @LaurenceLiZhixin Help wanted

Comment (LaurenceLiZhixin) [reply internal/message/message.go]: Yeah, MsgQueueIsBetter.

**Review by LaurenceLiZhixin — APPROVED** [review]

**Pull request merged**